README.md

```diff
@@ -38,11 +38,11 @@ Analyzer result summary: outdated=3, vulnerabilities=1, failed=0
 11
 ```
 
-Ok => 0  
```

Comment:
Please undo this change as it changes the contract of exit states.

Reply:
Hi,

So what's the contract of exit state ?

Reply:
> I'm not sure about the contracts of the install script, but if try to re-run the script, I get this:
> 
> ```
> [lukas] /tmp →  sh ./install.sh
> [INFO] wpscan-analyze install script for MacOS and Linux
> [INFO] wpscan-analyze in already installed: /Users/lukas/.cargo/bin/wpscan-analyze (wpscan-analyze 1.0.4)
> [INFO] The script will remove this installation
> [ERROR] You already have the latest wpscan-analyze version. If you're looking for the dev version please install it manually.
> ```
> 
> This seems odd, as the script says it's going to remove the current installation, but then bails.

I'll make the script allow re-install of the same version